src/views/texts/TextDetails.vue

```diff
@@ -111,6 +115,23 @@ export default class TextDetails extends Vue {
 				this.snackbarText = 'Error while adding text to the project';
 			});
 	}
+
+	private async removeText(): Promise<void> {
+		Vue.$axios.delete(`/project/${this.$route.query.project_id}/delete_text`, {
+				data: { text_id: this.masterId },
+			})
+			.then((response: AxiosResponse) => {
+				this.getTextDetails();
```

Comment:
What text details are you getting? First you already have them and second you shouldn't need them if your removing a project. Remember not to copy things unless you fully understand its functionality.